metaflow/plugins/kubernetes/kubernetes.py

```diff
@@ -178,7 +180,7 @@ def create_job(
         job = (
             KubernetesClient()
             .job(
-                generate_name="t-",
+                generate_name="t-{entropy}".format(entropy=secrets.token_hex(2)),
```

Comment:
`create_job` receives `secrets` as a parameter, so this won't work. Small change to directly import and use `token_hex` should fix it.

```python
from secrets import token_hex
```

Reply:
This won't work either if Metaflow still plans on supporting python 3.5.x

Running

```
mamba create -p /tmp/test 'python==3.5.5'
mamba activate /tmp/test
python -c "import secrets"
```

Results in module not found